samples/powerpoint/basics/get-slide-metadata.yaml

```diff
@@ -0,0 +1,56 @@
+id: ''
+gist: ''
+source: POWERPOINT
+author: Microsoft
+name: get-slide-metadata
+description: 'Gets the title, index, and ID of the selected slide(s).'
+script:
+  content: |-
+    $("#run").click(run);
+
+    function run() {
```

Comment:
This conflicts with the guidance in the Readme for wrathofzombies/samples:
"Each button-click handler should have its own async function, called "run" if there is only one button on the page "